nano/lib/config.hpp

```diff
@@ -251,6 +251,8 @@ class network_constants
 			telemetry_request_interval = 500ms;
 			telemetry_broadcast_interval = 500ms;
 			optimistic_activation_delay = 2s;
+			rep_crawler_normal_interval = 500ms;
+			rep_crawler_warmup_interval = 500ms;
```

Comment:
Since the rep crawler is now logging to debug, decreasing it will lead to more noise in logs. If there is a noticeable impact on unit test runtimes we may want to tune it, but I would keep this parameter conservative.